src/Document_Item.php

```diff
@@ -505,6 +505,8 @@ public static function showForDocument(Document $doc)
                     echo "<td class='center'>" .
                         (isset($data["otherserial"]) ? "" . $data["otherserial"] . "" : "-") . "</td>";
                     echo "</tr>";
+
+                    $item = getItemForItemtype($type_row['itemtype']);
```

Comment:
Could you explain this change ?

Reply:
This is to correct the display here
Before :
![image](https://github.com/user-attachments/assets/10504c49-7041-42a9-b203-be79a873a54b)

After:
![image](https://github.com/user-attachments/assets/afe99357-2c8b-4edb-9bcf-c98e2eac0365)